### PR TITLE
add TAP form styling

### PR DESCRIPTION
**Pull request by karenhawkinson**
Asana ticket: [Build CSS File for TAP](https://app.asana.com/0/1170867711449497/1202429475090094/f)

- Added new styling file for TAP
- Background color now shared across forms
- Custom components that are used on some forms but not others separated out into partials:
    - Photo grid
    - Red end state error messages (this changes Youth Pass errors slightly – the red border should now show on smaller screens)
    - Blue info message (new for TAP, expected to add to Youth)
- Added a new color variable for the background of the blue error message. (This color meets WCAG color contrast requirements, even when blue link text is used.)

### CODE REVIEW

File: stylesheets/tap.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@use "common" as *;
+@use "photo-grid" as *;
+@use "blue-alert" as *;
```
Comment (arkadyan) [inline stylesheets/tap.scss:3]: So much for free! 🎉

**Review by arkadyan — CHANGES_REQUESTED** [review]
Looking good. I left a couple of comments thinking about potential ways to continue to simplify.

File: stylesheets/_red-alert.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
+@use "common/colors" as *;
+
+/* Red error alert */
+.error-alert-item {
+  background-color: $form-error-message;
+  border-left: 0.25rem solid $error-text;
+  overflow: hidden;
```
[the remark this answers is not in the document]
Comment (karenhawkinson) [reply stylesheets/_red-alert.scss:7]: Agree, especially when these are added to `common` it doesn't make sense to separate these files.

**Review by arkadyan — APPROVED** [review]
👍

**Pull request merged**